dm/syncer/syncer.go

```diff
@@ -1248,7 +1250,8 @@ func (s *Syncer) successFunc(queueID int, jobs []*job) {
 		s.addCount(true, queueBucket, sqlJob.tp, 1, sqlJob.targetTable)
 	}
 	s.updateReplicationJobTS(nil, dmlWorkerJobIdx(queueID))
-	metrics.ReplicationTransactionBatch.WithLabelValues(s.cfg.WorkerName, s.cfg.Name, s.cfg.SourceID, queueBucket).Observe(float64(len(jobs)))
+	metrics.ReplicationTransactionBatch.WithLabelValues(s.cfg.WorkerName, s.cfg.Name, s.cfg.SourceID, queueBucket, "statements").Observe(float64(statementsCnt))
```

Comment:
statements means sql that really  exec to the downstream, rows means sql that come from binlog. am i right?
maybe need update the metrics doc https://docs.pingcap.com/zh/tidb-data-migration/stable/monitor-a-dm-cluster

Reply:
statements mean how many statements are in one transaction. rows mean how many rows changed in one transaction. I will update the doc later